public_api/public_api_basics.ipynb

```diff
@@ -0,0 +1,531 @@
+{
```

Comment:
<p>i think there may be a benefit in creating a separate print function from the search</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SystemInternal/SystemCookbook/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>agree with this mostly. I think we can return the df from this and then in the next block do the print functions of various parts of that data frame instead</p>